gen/icons/FlutterIcons.java

```diff
@@ -43,6 +43,7 @@ private static Icon load(String path) {
   public static final Icon CustomMethod = load("/icons/custom/method.png");
   public static final Icon CustomMethodAbstract = load("/icons/custom/method_abstract.png");
   public static final Icon CustomProperty = load("/icons/custom/property.png");
+  public static final Icon CustomInfo = load("/icons/custom/info.png");
```

Comment:
Used to render info icon that says D to denote properties matching their default value.